test/e2e/node/pod_resize.go

```diff
@@ -924,8 +1088,21 @@ var _ = SIGDescribe(framework.WithSerial(), "Pod InPlace Resize Container (defer
 	doPodResizeRetryDeferredTests(f)
 })
 
-var _ = SIGDescribe("Pod InPlace Resize Container", framework.WithFeatureGate(features.InPlacePodVerticalScaling), func() {
-	f := framework.NewDefaultFramework("pod-resize-tests")
+var _ = SIGDescribe("Pod InPlace Resize Container (resource-quota)", framework.WithFeatureGate(features.InPlacePodVerticalScaling), func() {
+	f := framework.NewDefaultFramework("pod-resize-resource-quota-tests")
+
+	ginkgo.BeforeEach(func(ctx context.Context) {
+		node, err := e2enode.GetRandomReadySchedulableNode(ctx, f.ClientSet)
+		framework.ExpectNoError(err)
+		if framework.NodeOSDistroIs("windows") || e2enode.IsARM64(node) {
+			e2eskipper.Skipf("runtime does not support InPlacePodVerticalScaling -- skipping")
+		}
+	})
+	doPodResizeResourceQuotaTests(f)
```

Comment:
is it possible to define the functions below where they're called? personal preference

Reply:
ah it seems the whole file is this way. nevermind

Reply:
you and I share the same personal preference :) but yeah didn't want to mess with the existing file structure